bots/tests-scan

```diff
@@ -78,7 +78,14 @@ REPO_BRANCH_CONTEXT = {
             'cockpit/fedora-30/vmware',
         ],
         '_manual': [
-            'cockpit/rhel-8-0',
+            'cockpit/rhel-8',
```

Comment:
There is no "rhel-8" image indeed, it needs to be explicitly "rhel-8-0" or "rhel-8-1".

Reply:
Updated and filed https://github.com/cockpit-project/cockpit/issues/12031 to discuss resolving this.